app/javascript/bundles/StarsRating/components/StarsRating.jsx

```diff
@@ -77,8 +77,8 @@ export default class StarsRating extends React.Component {
             rating: newRating,
             isHoverMode: false
         });
-        if (this.props.ratingCallback) {
-            this.props.ratingCallback(newRating);
+        if (this.props.onRating) {
+            this.props.onRating(newRating);
         }
```

Comment:
There's a nice way of avoiding a check like this; you can make `onRating` be a "no-op" function by default.
For example:
```js
static defaultProps = {
  onRating: function() {},
}
```

That way you don't have to check if it exists and just call it.

You can also define it in some `noop.js` file and export it, if you use it in more than one place.

There are a bunch of libraries, like Lodash, which provide a No-Op function that you can import, if you happen to already use one.
(Not that there's any real need to _import_ an empty function...)